source/Settleup.py

```diff
@@ -0,0 +1,87 @@
+import desert
+from group import Group, UserGroupView
+import pyrebase
+import requests
+import json
+import user_auth_config
+
+
+class Settleup:
+
+    def __init__(self):
+        config = {
+            "apiKey": "AIzaSyCfMEZut1bOgu9d1NHrJiZ7ruRdzfKEHbk",
+            "authDomain": "settle-up-sandbox.firebaseapp.com",
+            "databaseURL": "https://settle-up-sandbox.firebaseio.com",
+            "storageBucket": "settle-up-sandbox.appspot.com"
+        }
+
+        firebase = pyrebase.initialize_app(config)
+
+        auth = firebase.auth()
+        # user = auth.create_user_with_email_and_password("email@gmail.com", "password")
+        self.user = auth.sign_in_with_email_and_password(user_auth_config.email, user_auth_config.password)
+
+        # before the 1 hour expiry:
+        self.user = auth.refresh(self.user['refreshToken'])
+        # now we have a fresh token
+        self.user['idToken']
+
+        self.payload = {'auth': self.user['idToken']}
+
+    def get_groups(self):
+        # usergroup_json = api.request("userGroups")
+        r = requests.get('https://settle-up-sandbox.firebaseio.com/userGroups/' + str(self.user['userId'] + '.json'),
+                         params=self.payload)
```

Comment:
Line is too long
Should use fstrings
All request stuff should be abstracted out of the individual methods